apps/condo/domains/organization/integrations/sbbol/SbbolSecretStorage.js

```diff
@@ -38,31 +38,43 @@ class SbbolSecretStorage {
         await this.#setValue('clientSecret:updatedAt', dayjs().toISOString())
     }
 
-    async getAccessToken () {
-        return this.#getValue('accessToken')
+    async getAccessToken (userId) {
+        return this.#getValue(`user:${userId}:accessToken`)
     }
 
-    async setAccessToken (value, options) {
-        await this.#setValue('accessToken', value, options)
-        await this.#setValue('accessToken:updatedAt', dayjs().toISOString())
+    async setAccessToken (
+        value,
+        userId,
+        options,
+    ) {
+        if (!value ||
+            !userId
+        ) return logger.error('value and userId is required for setAccessToken')
```

Comment:
One condition for one variable, please. This way it's easier to figure out where exactly the problem occurs
Throw error, instead of logger, please